src/scripts/main.js

```diff
@@ -1,3 +1,50 @@
 'use strict';
 
-// write code here
+const employees = document.querySelectorAll('li');
+const len = employees.length;
+
+const massSalary = [];
+const massObjPeople = [];
+
+for (let i = 0; i < len; i++) {
+  let int = employees[i].getAttribute('data-salary').replace('$', '');
+
+  int = +int.replace(',', '');
+
+  massSalary.push(int);
+}
+
+for (let i = 0; i < len; i++) {
+  const objPeople = {};
+
+  let int = employees[i].getAttribute('data-salary').replace('$', '');
+
+  int = +int.replace(',', '');
+
+  objPeople.name = employees[i].textContent;
+  objPeople.position = employees[i].getAttribute('data-position');
+  objPeople.salary = int;
+  objPeople.age = employees[i].getAttribute('data-age');
+
+  massObjPeople.push(objPeople);
+}
+
+massSalary.sort((a, b) => b - a);
+
+function getEmployees(list) {
+  return massObjPeople;
```

Comment:
The `getEmployees` function is defined to return `massObjPeople`, but it is called without using its return value. Consider using the return value or removing the function if it's not needed.